examples/lstm_text_generation.py

```diff
@@ -127,16 +127,16 @@ def main(num_epochs=NUM_EPOCHS):
 
     l_forward_2 = lasagne.layers.LSTMLayer(
         l_forward_1, N_HIDDEN, grad_clipping=GRAD_CLIP,
-        nonlinearity=lasagne.nonlinearities.tanh)
+        nonlinearity=lasagne.nonlinearities.tanh,
+        only_return_final=True)
 
-    # The l_forward layer creates an output of dimension (batch_size, SEQ_LENGTH, N_HIDDEN)
-    # Since we are only interested in the final prediction, we isolate that quantity and feed it to the next layer. 
-    # The output of the sliced layer will then be of size (batch_size, N_HIDDEN)
-    l_forward_slice = lasagne.layers.SliceLayer(l_forward_2, -1, 1)
+    # A SliceLayer is no longer required, since l_forward layer creates an output of dimension (batch_size, N_HIDDEN) because
```

Comment:
If someone reads this code, they will wonder what `no longer` refers to. Also the next comment starts with `The sliced output`, which does not fit the code any more. Can you amend your commit to just drop the `SliceLayer` code and comment, and adapt the next comment?